js/battledata.js

```diff
@@ -848,7 +848,7 @@ var Tools = {
 		}
 
 		var animationData = {};
-		if (options.gen === 5 && window.BattlePokemonSpritesBW) {
+		if ( (options.gen === 5 || Tools.prefs('bwgfx') ) && window.BattlePokemonSpritesBW && !Tools.prefs('nopastgens')) {
```

Comment:
I have them because I find spaces make it easier to pair up each pair of parentheses in multiconditional statements like `(options.gen === 5 || Tools.prefs('bwgfx')`. If PS do things without those, I'll remove them.